js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
+  if (trash) {
+    return;
+  }
+
+  //Presenting Todays date in task list inside li tag
+  const options = {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  };
+
+  tarikh = today.toLocaleDateString("en-us", options);
+  const item = `<li class="item">                  
+                  <p class="text">${toDo}</p>
+                  <!-- data presenter div tag with date class  -->
+                  <div class = "date" id = "${id}"> Created At:  ${tarikh} </div>
+                  <img src= "icons/done.svg" job = "${COMPLETE}" id ="${id}"/>
+                  <img src = "icons/edit.svg" job = "${EDIT}" id = "${id}"/>
+                  <img src = "icons/delete.svg" job="${DELET}" id="${id}"/>
+                  </li> `;
+
+  const position = "afterbegin";
+  list.insertAdjacentHTML(position, item);
+}
+
+createTaskButton.addEventListener(CLICK_EVENT, function (event) {
+  if (inputTask.className == "hide") {
```

Comment:
```suggestion
  if (inputTask.className === "hide") {
```
Use strict equality: Instead of using loose equality (==) to compare class names, use strict equality (===) to avoid unexpected behavior.

Reply:
Since it's best practice, I have updated the comparison operator with a strict equality comparison operator.